src/components/Footer/index.jsx

```diff
@@ -1,64 +1,48 @@
 import React from "react"
 
-import CallToAction from "~/src/components/CallToAction"
+import NewsletterCTA from "../NewsletterCTA"
 import Text from "~/src/components/Text"
-import Logo from "~/src/components/Logo"
-import Location from "./location"
-import Norte2020 from "./Norte2020"
+import Link from "~/src/components/Link"
 import SocialLinks from "./social_links"
 
 import * as styles from "./index.module.css"
 
 function Footer() {
   return (
-    <footer>
-      <div className={styles.blue}>
-        <div className={styles.content}>
-          <div className={styles.logo}>
-            <Logo />
+    <footer className={styles.container}>
+      <NewsletterCTA />
+      <div className={styles.content}>
```

Comment:
Let's make a separate component for this part. Something like `FooterGrid`, then you could have a hierarchy like this.

`root` > `item` > `label`